app/js/places/places.controller.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+    'use strict';
+    angular.module('cityInfo.places').controller('PlacesController', PlacesController);
+
+    PlacesController.$inject = ['$scope'];
+
+    function PlacesController ($scope) {
+        $scope.POIlist= [{
```

Comment:
It would be better to not store this list in the scope, but to the controller, something like:
var vm = this;
vm.POIlist = [*stuff*];
And then in the html you can access the poilist like pc.POIlist